Notifier-service/package.json

```diff
@@ -4,23 +4,28 @@
   "description": "Notification service for Conference-Notify",
   "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "cross-env NODE_ENV=test jest --testTimeout=10000",
     "build": "tsc",
     "start": "ts-node-dev --respawn src/server.ts",
     "run": "node build/server.js"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "body-parser": "^1.19.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "jest": "^25.1.0",
     "mongoose": "^5.9.3",
+    "sqlite3": "^4.1.1",
+    "supertest": "^4.0.2",
```

Comment:
* Why needed ?

Reply:
While testing for unit tests, I tried using this frameworks. Sorry, I will remove it.